.pylintrc

```diff
@@ -0,0 +1,67 @@
+[MESSAGES CONTROL]
+disable=
+# "F" Fatal errors that prevent further processing
+ import-error,
+# "I" Informational noise
+# "E" Error for important programming issues (likely bugs)
+ cyclic-import,
+ no-member,
+ no-name-in-module,
+ undefined-variable,
+# "W" Warnings for stylistic problems or minor programming issues
+ abstract-method,
+ arguments-differ,
+ bad-mcs-classmethod-argument,
+ broad-except,
+ dangerous-default-value,
+ deprecated-lambda,
+ eval-used,
+ exec-used,
+ expression-not-assigned,
+ fixme,
+ function-redefined,
+ global-statement,
+ not-callable,
+ protected-access,
+ redefined-builtin,
+ redefined-outer-name,
+ super-init-not-called,
+ unsupported-membership-test,
+ unsubscriptable-object,
+ unused-argument,
+ unused-variable,
+ useless-super-delegation,
+# "C" Coding convention violations
+ bad-continuation,
+ consider-iterating-dictionary,
+ invalid-name,
+ len-as-condition,
+ missing-docstring,
+ superfluous-parens,
+ unidiomatic-typecheck,
+ unnecessary-lambda,
+# "R" Refactor recommendations
+ consider-using-dict-comprehension,
+ duplicate-code,
+ inconsistent-return-statements,
+ redefined-argument-from-local,
+ no-else-return,
+ no-else-raise,
+ too-few-public-methods,
+ too-many-ancestors,
+ too-many-arguments,
+ too-many-branches,
+ too-many-instance-attributes,
+ too-many-locals,
+ too-many-public-methods,
+ too-many-return-statements,
+ too-many-statements,
+ trailing-comma-tuple,
+# new for python3 version of pylint
+ consider-using-set-comprehension,
+ unnecessary-pass,
+ useless-object-inheritance
+
+[FORMAT]
+# Maximum number of characters on a single line.
+max-line-length=120
```

Comment:
Maybe change this to 119 to be consistent with [leapp-repository](https://github.com/oamg/leapp-repository/pull/211#pullrequestreview-242207314)?

Reply:
I left 120 on purpose as changing it to 119 instantly shows that existing code base has violated this rule many times. imho 120->119 is a change for the next patch.
`=================================== FAILURES ===================================
______________________ [pylint] leapp/actors/__init__.py _______________________
C:308, 0: Line too long (120/119) (line-too-long)
C:313, 0: Line too long (120/119) (line-too-long)
C:324, 0: Line too long (120/119) (line-too-long)
C:355, 0: Line too long (120/119) (line-too-long)
C:378, 0: Line too long (120/119) (line-too-long)
_______________________ [pylint] leapp/dialogs/dialog.py _______________________
C: 55, 0: Line too long (120/119) (line-too-long)
___________________ [pylint] leapp/models/fields/__init__.py ___________________
C:447, 0: Line too long (120/119) (line-too-long)
____________________ [pylint] leapp/repository/__init__.py _____________________
C:  9, 0: Line too long (120/119) (line-too-long)
_________________ [pylint] leapp/snactor/commands/discover.py __________________
C:102, 0: Line too long (120/119) (line-too-long)
C:123, 0: Line too long (120/119) (line-too-long)
_______________________ [pylint] leapp/tags/__init__.py ________________________
C: 24, 0: Line too long (120/119) (line-too-long)
___________________________ test_audit_command_in_db ___________`